src/ServiceInsight/MessageFlow/MessageActionPopup.xaml

```diff
@@ -17,7 +17,7 @@
                     <SolidColorBrush x:Key="GlyphBrush" Color="#444" />
                     <SolidColorBrush x:Key="DisabledForegroundBrush" Color="#888" />
                     <SolidColorBrush x:Key="WindowBackgroundBrush" Color="#000" />
-                    <SolidColorBrush x:Key="SelectedBackgroundBrush" Color="#007AC7" />
+                    <SolidColorBrush x:Key="SelectedNodeBackgroundBrush" Color="#007AC7" />
```

Comment:
There is another brush called SelectedBackgroundBrush which is also in the scope and was conflicting with this. They have different values though so I have changed the name of this one here.